packages/rancher-windows-gmsa/templates/crd-template/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+version: 1.0.0
+description: Installs the CRDs for Windows GMSA.
+name: rancher-windows-gmsa-crd
+type: application
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: kube-system

```

Comment:
Is there a reason why this is in `kube-system`? Generally, we place feature charts in their own namespaces, e.g. `cattle-monitoring-system` for Monitoring or `cattle-windows-gmsa-system` for example

Reply:
this is also the pattern for wins-upgrader: https://github.com/rancher/charts/blob/502035b9ac9763e2ebbdde5c05f2137862bb2678/charts/rancher-wins-upgrader/100.0.0%2Bup0.0.1/Chart.yaml#L3